### PR TITLE
No fail fast and secrets passing to subflow

### PR DESCRIPTION
**Pull request by TomaszGaweda**

### CODE REVIEW

File: .github/workflows/vulnerability_scan_subworkflow.yml [hunk 1]
```diff
@@ -11,12 +11,15 @@ on:
             ref:
                 required: true
                 type: string
+        secrets:
+            SNYK_TOKEN:
+                required: true
```
Comment (ldziedziul) [inline .github/workflows/vulnerability_scan_subworkflow.yml:16]: could we use `${{ secrets.SNYK_TOKEN }}` for `workflow_dispatch` ?

Comment (TomaszGaweda) [reply .github/workflows/vulnerability_scan_subworkflow.yml:16]: not sure about it, this method is mentioned in GitHub docs

Comment (zhmaeff) [reply .github/workflows/vulnerability_scan_subworkflow.yml:16]: https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow
AFAICS, we can use secrets param only for `workflow_call`.
For `workflow_dispatch`, secrets are available as per org/repo settings.

**Review by ldziedziul — APPROVED** [review]
Nice job!

**Pull request merged**